src/main/java/org/folio/marccat/integration/AuthorityStorageService.java

```diff
@@ -192,11 +210,10 @@ private CatalogItem insertAuthorityRecord(final AuthorityRecord record, final in
     item.getItemEntity().setLanguageOfCataloguing(lang);
```

Comment:
Edit and add missing @param to the method for javadoc

Reply:
I have changed it